homeassistant/components/openerz/sensor.py

```diff
@@ -58,9 +59,25 @@ def native_value(self):
         """Return the state of the sensor."""
         return self._state
 
+    @property
+    def device_class(self):
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return SensorDeviceClass.TIMESTAMP
+
     def update(self):
         """Fetch new state data for the sensor.
 
         This is the only method that should fetch new data for Home Assistant.
+
+        The api returns a string of format "%Y-%m-%d". From the ERZ website,
+        it can be deducted that the pickup happens after 07:00 in the morning.
+
+        As this platform only works for Zurich, the timezone will be set to Zurich.
         """
-        self._state = self.api_connector.find_next_pickup(day_offset=31)
+        date_string = self.api_connector.find_next_pickup(day_offset=31)
+        time_string = "07:00"
+        datetime_string = f"{date_string} {time_string}"
```

Comment:
Not sure this needs to be that verbose... Maybe you can just make the actual time directly a part of the `datetime_string` as it's not used anywhere else?